src/Common.jl

```diff
@@ -1,10 +1,4 @@
 
-import Base: *
-
-
-# FIXME remove! is it really needed? This is type piracy
-*(a::Symbol, b::AbstractString)::Symbol = Symbol(string(a,b))
```

Comment:
Yes, i use this -- not quite sure i see what is wrong with doing so?

Reply:
https://docs.julialang.org/en/v1/manual/style-guide/index.html#Avoid-type-piracy-1

Reply:
okay, leave it deleted then -- this case would probably not have caused too many problems but will toe the Julia line. thanks

Reply:
Note, i'ts not used in DFG at all.